prover-sever/src/mock_client.rs

```diff
@@ -0,0 +1,92 @@
+pub mod prove;
+pub mod spec;
+pub mod utils;
+use crate::prove::ProofResult;
+use crate::spec::{ProofType, ZkSpec};
+use crate::utils::kroma_info;
+use clap::Parser;
+use jsonrpsee::http_client::{HttpClient, HttpClientBuilder};
+use jsonrpsee_core::client::ClientT;
+use jsonrpsee_core::rpc_params;
+use std::fs;
+use std::time::Duration;
+use types::eth::BlockTrace;
```

Comment:
Please sort them.

Reply:
The "cargo fmt" already sort them (maybe it likes the order).
By the way, I have added a blank line to distinguish between module imports and package imports.